src/table_compaction.cc

```diff
@@ -28,8 +28,8 @@ namespace jungle {
 //
 Status TableMgr::compactLevelItr(const CompactOptions& options,
                                  TableInfo* victim_table,
-                                 size_t level)
-{
+                                 size_t level,
+                                 bool adjusting_num_l0) {
```

Comment:
If passing `adjusting_num_l0` is for blocking compaction cancellation, we should remove it.